appstudio-controller/controllers/appstudio.redhat.com/promotionrun_controller.go

```diff
@@ -23,6 +23,7 @@ import (
 	"time"
 
 	"github.com/go-logr/logr"
+	applicationv1alpha1 "github.com/redhat-appstudio/application-service/api/v1alpha1"
```

Comment:
https://github.com/redhat-appstudio/managed-gitops/pull/261 is replacing application-service repository with appplication-api. You will most likely need to change this.